### PR TITLE
LSR-979 Cookie login (iOS)

### PR DESCRIPTION
**Pull request by victorg1991**
This PR uses the latest changes of https://github.com/liferay/liferay-mobile-sdk

To test it you need:
 
- set the inspectable property loginMode in LoginScreenlet to "cookie"

 - add to the Showcase Podfile: pod 'Liferay-iOS-SDK', :branch => 'master'

### CODE REVIEW

File: ios/Framework/Core/Context/LRCookieBlockCallback.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//
+//  LRCookieBlockCallback.swift
+//  LiferayScreens
+//
+//  Created by Victor Galán on 13/02/2017.
```
Comment (dgarciasarai) [inline ios/Framework/Core/Context/LRCookieBlockCallback.swift]: Copyright text plz :)

**Comment by dgarciasarai** [comment]
LGTM 👍  Make sure that new files have correct copyright text. Great job!

File: ios/Framework/Core/Context/LRCookieBlockCallback.swift (hunk 1 above)
Comment (victorg1991) [inline ios/Framework/Core/Context/LRCookieBlockCallback.swift]: Done 👍

**Comment by nhpatt** [comment]
👍

**Pull request merged**